modules/ossm-vs-istio-1x.adoc

```diff
@@ -90,7 +90,7 @@ spec:
 
 OpenShift routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an OpenShift route is created, updated or deleted.
 
-A {ProductName} control plane component called Istio OpenShift Routing (IOR) synchronizes the gateway route.  For more information see the "Automatic route creation" section.
+A {ProductName} control plane component called Istio OpenShift Routing (IOR) synchronizes the gateway route.  For more information, see xref:../../service_mesh/v1x/ossm-traffic-manage.adoc#ossm-auto-route_routing-traffic-v1x[Automatic route creation].
```

Comment:
You can't put a cross-reference in a module.

Reply:
@JStickler I thought they made an update to pantheon that cross-reference in a module are supported. It looks like it works in preview.

Reply:
Yea, I took another look and removed the links.